# ==============================================================================
# REGRESSOR COMMIT DIFF
# ==============================================================================
# File: js/src/wasm/WasmTlsData.h
# Commit: ce799cb566ac
# Full Hash: ce799cb566ac0fb95468118608132be11db6a359
# Author: Ryan Hunt <rhunt@eqrion.net>
# Date: 2022-03-24 21:54:20
# Regressor Bug: 1642412
# File Overlap Count: 19
# Description:
#   Bug 1642412 - wasm: Rollup of inline table.get/set for anyref. r=lth
#   
#   Rollup of the previous inline table.get/set patches.
#   
#   Differential Revision: https://phabricator.services.mozilla.com/D141024
# ==============================================================================

diff -r c10609d434ae -r ce799cb566ac js/src/wasm/WasmTlsData.h
--- a/js/src/wasm/WasmTlsData.h	Thu Mar 24 17:40:32 2022 +0000
+++ b/js/src/wasm/WasmTlsData.h	Thu Mar 24 17:56:50 2022 +0000
@@ -73,7 +73,8 @@
 
   // Pointer to the array of elements (which can have various representations).
   // For tables of anyref this is null.
-  void* functionBase;
+  // For tables of functions, this is a pointer to the array of code pointers.
+  void* elements;
 };
 
 // Table element for TableRepr::Func which carries both the code pointer and
